easyDiffractionApp/Gui/Pages/Analysis/SideBarAdvanced.qml

```diff
@@ -83,47 +126,207 @@ EaComponents.SideBarColumn {
+                        model: XmlListModel {
+                            xml: ExGlobals.Constants.proxy.parametersAsXml
+                            query: "/root/item"
+                            XmlRole { name: "label"; query: "label_with_index/string()" }
+                            XmlRole { name: "iconified_label"; query: "iconified_label_with_index/string()" }
+                            onXmlChanged: dependentParCurrentIndex = dependentPar.currentIndex
+                        }
+                        onCurrentIndexChanged: {
+                            //print(currentText)
+                            if (dependentPar.currentIndex === -1 && model.count > 0)
+                                dependentPar.currentIndex = dependentParCurrentIndex
+                        }
+                    }
+
+                    EaElements.ComboBox {
+                        id: relationalOperator
+                        width: 47
+                        currentIndex: 0
+                        font.family: EaStyle.Fonts.iconsFamily
+                        //model: ["=", ">", "<"]
+                        model: XmlListModel {
+                            xml: "<root><item><operator>=</operator><icon>\uf52c</icon></item><item><operator>&gt;</operator><icon>\uf531</icon></item><item><operator>&lt;</operator><icon>\uf536</icon></item></root>"
+                            query: "/root/item"
+                            XmlRole { name: "icon"; query: "icon/string()" }
+                        }
+                    }
+
+                    Item { height: 1; width: 1 }
+                    Item { height: 1; width: 1 }
+
+                    EaElements.ComboBox {
+                        id: independentPar
+                        width: dependentPar.width
+                        currentIndex: -1
+                        displayText: currentIndex === -1 ? "Select parameter" : currentText
+                        textFormat: ExGlobals.Variables.iconifiedNames ? Text.RichText : Text.PlainText
+                        elide: Text.ElideMiddle
+                        textRole: ExGlobals.Variables.iconifiedNames ? "iconified_label" : "label"
+                        model: XmlListModel {
+                            xml: ExGlobals.Constants.proxy.parametersAsXml
+                            query: "/root/item"
+                            XmlRole { name: "label"; query: "label_with_index/string()" }
+                            XmlRole { name: "iconified_label"; query: "iconified_label_with_index/string()" }
+                            onXmlChanged: independentParCurrentIndex = independentPar.currentIndex
+                        }
+                        onCurrentIndexChanged: {
+                            if (independentPar.currentIndex === -1 && model.count > 0)
+                                independentPar.currentIndex = independentParCurrentIndex
+                        }
+                    }
+
```

Comment:
`dependentPar2`, `dependentPar` and `independentPar` comboboxes share most of their layout and it would be good to make a common base.
Similarly for the arithmetic operators.